ci/scripts/run-test-network-ledger.sh

```diff
@@ -14,8 +13,28 @@ function print() {
 print "Creating network"
 ./network.sh up createChannel -ca -s couchdb -i "${FABRIC_VERSION}"
 
-print "Deploying ${CHAINCODE_NAME} chaincode"
-./network.sh deployCC -ccn "${CHAINCODE_NAME}" -ccv 1 -ccs 1 -ccl "${CHAINCODE_LANGUAGE}"
+print "Deploying ${CHAINCODE_NAME} go chaincode"
+./network.sh deployCC -ccn "${CHAINCODE_NAME}" -ccv 1 -ccs 1 -ccl go
+
+# Run Javascript application against the go chaincode
+print "Initializing Javascript application"
+pushd ../asset-transfer-ledger-queries/application-javascript
+npm install
+print "Executing app.js"
+node app.js
+popd
+
+print "Deploying ${CHAINCODE_NAME} javascript chaincode"
+./network.sh deployCC -ccn "${CHAINCODE_NAME}" -ccv 2 -ccs 2 -ccl javascript
```

Comment:
I think using ccv 2 , and the sequence 2 will fail, since the lifecycle expects sequence 1 for each version, no?

better to add it to the azure matrix with a chaincode_language of javascript & go, as in
https://github.com/hyperledger/fabric-samples/blob/master/ci/azure-pipelines.yml#L152-L160

Reply:
Thanks, I have changed the version to be more of a version, like 1.0, 2.0 and not just an integer, which is why we separated the two into the two settings (-ccv and -ccs). The version can be any string, however the sequence must be an integer and must begin at one and be incremented with each change to the chaincode. The fabric-sample's helper script (deployCC) only provides changing all things at the same time, code and policies, therefore we need to change both these and be sure increment the sequence.

Reply:
thanks for explaining